Makefile

```diff
@@ -0,0 +1,8 @@
+.PHONY: develop test
+
```

Comment:
I'm not really a fan of Makefiles personally. Is it possible we could just reuse the existing `runtests.py` file? Otherwise, it's not that big of a deal I guess